.changeset/dull-spies-love.md

```diff
@@ -0,0 +1,18 @@
+---
+'@talend/react-faceted-search-query-client': major
+'@talend/react-faceted-search': major
+---
+
+[BREAKING] Move the function createTqlQuery to a dedicated package (@talend/react-faceted-search-query-client).
+
+You will need to adapt your code to use this lib
+
+before
+```
+import { createTqlQuery } from '@talend/react-faceted-search/lib/queryClient';
+```
+
+after
+```
+import { createTqlQuery } from '@talend/react-faceted-search-query-client'
```

Comment:
```suggestion
import { createTqlQuery } from '@talend/react-faceted-search-query-client'
import * as queryClient from '@talend/react-faceted-search-query-client'
```

Reply:
Not sure for this one as there is not default export.

Reply:
`import *` is not about default but about getting all named export under the queryClient object